김진/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,18 @@
+function solution(s) {
+  upper_s = s.toUpperCase();
+  let pNum = 0;
+  let yNum = 0;
+  for (let i = 0; i < upper_s.length; i++) {
+    if (upper_s[i] == "P") {
+      pNum++;
+    } else if (upper_s[i] == "Y") {
+      yNum++;
+    }
+  }
+
+  if (pNum == yNum || (pNum == 0 && yNum == 0)) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
```suggestion
    return true;
  }
  return false;

```
사실 코드적으로 문제는 없는데 더 좋은 패턴 추천드립니다.
(어차피 다른 조건이 필요 없이 true 아니면 무조건 false가 return 되기 때문)